emojis.json

```diff
@@ -2819,11 +2819,306 @@
+    "char": "🇧🇷",
+    "category": "flags"
+  },
+  "io": {
+    "keywords": ["british", "indian", "ocean", "territory", "flag", "nation", "country", "banner"],
+    "char": "🇮🇴",
+    "category": "flags"
+  },
+  "vg": {
+    "keywords": ["british", "virgin", "islands", "bvi", "flag", "nation", "country", "banner"],
+    "char": "🇻🇬",
+    "category": "flags"
+  },
+  "bn": {
+    "keywords": ["brunei", "darussalam", "flag", "nation", "country", "banner"],
+    "char": "🇧🇳",
+    "category": "flags"
+  },
+  "bg": {
+    "keywords": ["bulgaria", "flag", "nation", "country", "banner"],
+    "char": "🇧🇬",
+    "category": "flags"
+  },
+  "bf": {
+    "keywords": ["burkina", "faso", "flag", "nation", "country", "banner"],
+    "char": "🇧🇫",
+    "category": "flags"
+  },
+  "bi": {
+    "keywords": ["burundi", "flag", "nation", "country", "banner"],
+    "char": "🇧🇮",
+    "category": "flags"
+  },
+  "cv": {
+    "keywords": ["cabo", "verde", "flag", "nation", "country", "banner"],
+    "char": "🇨🇻",
+    "category": "flags"
+  },
+  "kh": {
+    "keywords": ["cambodia", "flag", "nation", "country", "banner"],
+    "char": "🇰🇭",
+    "category": "flags"
+  },
+  "cm": {
+    "keywords": ["cameroon", "flag", "nation", "country", "banner"],
+    "char": "🇨🇲",
+    "category": "flags"
+  },
+  "ca": {
+    "keywords": ["canada", "flag", "nation", "country", "banner"],
+    "char": "🇨🇦",
+    "category": "flags"
+  },
+  "ic": {
+    "keywords": ["canary", "islands", "flag", "nation", "country", "banner"],
+    "char": "🇮🇨",
+    "category": "flags"
+  },
+  "ky": {
+    "keywords": ["cayman", "islands", "flag", "nation", "country", "banner"],
+    "char": "🇰🇾",
+    "category": "flags"
+  },
+  "cf": {
+    "keywords": ["central", "african", "republic", "flag", "nation", "country", "banner"],
+    "char": "🇨🇫",
+    "category": "flags"
+  },
+  "td": {
+    "keywords": ["chad", "flag", "nation", "country", "banner"],
+    "char": "🇹🇩",
+    "category": "flags"
+  },
+  "chile": {
+    "keywords": ["flag", "nation", "country", "banner"],
+    "char": "🇨🇱",
+    "category": "flags"
+  },
   "cn": {
     "keywords": ["china", "chinese", "prc", "flag", "country", "nation", "banner"],
     "char": "🇨🇳",
     "category": "flags"
   },
+  "cx": {
+    "keywords": ["christmas", "island", "flag", "nation", "country", "banner"],
+    "char": "🇨🇽",
+    "category": "flags"
+  },
+  "cc": {
+    "keywords": ["cocos", "keeling", "islands", "flag", "nation", "country", "banner"],
+    "char": "🇨🇨",
+    "category": "flags"
+  },
+  "co": {
+    "keywords": ["colombia", "flag", "nation", "country", "banner"],
+    "char": "🇨🇴",
+    "category": "flags"
+  },
+  "km": {
+    "keywords": ["comoros", "flag", "nation", "country", "banner"],
+    "char": "🇰🇲",
+    "category": "flags"
+  },
+  "cg": {
+    "keywords": ["congo", "flag", "nation", "country", "banner"],
+    "char": "🇨🇬",
+    "category": "flags"
+  },
+  "drc": {
```

Comment:
Most of the country flags use the ISO 3166-2 country code as the emoji code. The ISO code to use here would be "cd", but ["cd" is already used by one of the previously-existing emojis](https://github.com/jasonrudolph/emojilib/blob/b0ce5ed37f3fefb42c4800f4398f430f9cc9392f/emojis.json#L3262), so I'm proposing that we use "drc" as the emoji code here, since that [seems to be a commonly-used abbreviation](https://en.wikipedia.org/wiki/Democratic_Republic_of_the_Congo).